### ISSUE TITLE
Can epic2-df support biological replicates?

### ISSUE THREAD

**Issue by hdshu** [biocore-ntnu/epic2#65]
Dear Sir , I want to use epic2-df  to call diffpeaks and  my each sample has two replicates. It seems that epic2-df  can not support two input Bams for one treatment or control.  Do you have any suggestion for this problem?  

Thank you very much! 

Best wishes!

**Comment by endrebak** [comment]
No, this was just a port of SICERs functionality. I'm not savvy about the
statistics
